bigquery/tables.js

```diff
@@ -252,7 +266,7 @@ cli
   .command('create <dataset> <table>', 'Create a new table in the specified dataset.', {}, function (options) {
     program.createTable(utils.pick(options, ['dataset', 'table']), utils.makeHandler());
   })
-  .command('list <dataset>', 'List tables in the specified dataset.', {}, function (options) {
+  .command('tables <dataset>', 'List tables in the specified dataset.', {}, function (options) {
```

Comment:
Or

```
node tables list my_dataset
node tables rows my_dataset my_table
or
node tables getRows my_dataset my_table
or
node tables get-rows my_dataset my_table
or
node tables listRows my_dataset my_table
or
node tables list-rows my_dataset my_table
```

I don't really mind what the CLI command is for running `listRows`, but I think the command for listing tables should be `list` for consistency.